totalRP3_Extended/dialogues/dialogues.lua

```diff
@@ -301,7 +304,7 @@ local function setModel(model, dialogStepClass, field)
 end
 
 -- Prepare all the texts for a step
-function processDialogStep()
+function processDialogStep(isNewDialog)
```

Comment:
Is this new parameter actually having a value passed in from somewhere else that isn't visible from this set of changes alone, or is it going to be perpetually nil?